### PR TITLE
move filesystem declaration after headers

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/util/process_executor.h [hunk 1]
```diff
@@ -39,11 +39,15 @@
 #include <filesystem>
 #include <unistd.h>
 
+namespace fs = std::filesystem;
+
 #include "executor.h"
 
+namespace fs = std::filesystem;
```
Comment (KarlStraussberger) [inline src/util/process_executor.h:46]: Why duplicate it?

File: src/cds_objects.h [hunk 2]
```diff
@@ -36,7 +36,6 @@
 #include <map>
 #include <memory>
 #include <vector>
-namespace fs = std::filesystem;
```
Comment (KarlStraussberger) [inline src/cds_objects.h:39]: If the alias is not needed, also the include must be in one of the others. Did you try to remove it as well?

**Review by KarlStraussberger — COMMENTED** [review]
I recommend a new header `utils/grb_fs.h` that encapsulates the include and the alias. We may move some of the declaration from tools.h/.cc to that file.

**Comment by neheb** [comment]
wouldn't common.h make more sense?

**Comment by KarlStraussberger** [comment]
> 
> 
> wouldn't common.h make more sense?

IHMO this would blow up. We should stick to include what we need.

**Comment by KarlStraussberger** [comment]
Thank you

**Pull request merged**